### DISCUSSION TITLE
setting my own theme name and textdomain

### DISCUSSION THREAD

**Discussion by yoscbd** [holger1411/understrap#1638] [Q&A]
Is there a way to chose my own textdomain when installing the theme npm package (a setting i can add on the package.json file before running NPM Install command)?

If not, is there a recommended way to replace "understrap" with my own textdomain at once? 
Thank you!

**Comment by bacoords** [comment]
Unfortunately not. The best option is probably to do a search/replace for `'understrap'`, keeping the quotes around it.
